src/commands/githook.ts

```diff
@@ -47,6 +62,7 @@ export const hookCommand = command(
           );
         }
 
+        console.log("made it")
```

Comment:
@matt-degraffenreid I think you forgot to remove that log
```suggestion

```

Reply:
hate when that happens, good eye. just update that.